assets/css/custom.css

```diff
@@ -1896,4 +1905,566 @@ a.read-all-posts:hover {
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-size: contain;
+  content: "";
+  height: 1.2rem;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: .25s;
+  width: 1rem;
+}
+
+@media (min-width:1200px) {
+  .new-header .main_menu>li.another_link a[target=_blank]:before {
+    height: 1.4rem;
+    width: 1.25rem;
+  }
+}
+
+.new-header .links_btns {
+  margin-left: 1.5rem;
+  padding-bottom: 26px;
+  transition: .25s;
+}
+
+@media (max-width:880px) {
+  .new-header .links_btns {
+    display: none;
+  }
+}
+
+.new-header .links_btns a {
+  display: inline-block;
+  width: 2rem;
+}
+@media (min-width:1200px) {
+  .new-header .links_btns a {
+    width: 2.7rem;
+  }
+}
+.new-header .links_btns a:first-of-type {
+  margin-right: 0.6875rem;
+}
+
+.new-header .links_btns a svg {
+  width: 100%;
+}
+.new-header .links_btns a:hover svg path {
+  fill: #0dddd3;
+}
+
+.new-header .links_btns a svg path {
+  transition: fill .25s;
+  width: 100%;
+}
+
+
+/* Mobile Nav */
+.new-header .mobile_menu {
+  background-color: #fff;
+  box-shadow: 0.25rem 0.25rem 3.125rem rgba(0,0,0,.25);
+  display: none;
+  height: 100vh;
+  max-width: 24rem;
+  opacity: 0;
+  overflow: auto;
+  padding: 1rem 1rem 5rem;
+  position: absolute;
+  right: 0;
+  top: 78px;
+  transform: translateX(100%),scaleX(0);
+  transition: transform .4s cubic-bezier(.77,.2,.05,1),opacity .4s ease,visibility .4s ease;
+  visibility: hidden;
+  width: 80vw;
+  z-index: 15;
+
+  opacity: 1;
+  transform: translateX(0),scaleX(1);
+  visibility: visible;
+  padding: 1.8rem 1.8rem 8rem;
+}
+
+@media (max-width:880px) {
+  .new-header .mobile_menu {
+    display: none;
+  }
+  .new-header .mobile_menu.is_active {
+    display: block;
+  }
+}
+
+.new-header .mobile_menu .main_menu {
+  opacity: 1;
+  transform: translateX(0),scaleX(1);
+  visibility: visible;
+  display: block;
+}
+
+.new-header .mobile_menu .main_menu>li  {
+  display: block;
+}
+
+.new-header .mobile_menu .main_menu>li>button {
+  border-bottom: none;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 19px;
+  padding-bottom: .9rem;
+}
+
+@media screen and (max-width: 880px) {
+  .new-header .mobile_menu .main_menu>li>button:hover {
+    border-bottom: 2px solid;
+    border-bottom-color: #0dddd3;
+  }
+}
+
+.new-header .mobile_menu .main_menu>li:hover .sub-menu {
+  margin: .8rem;
```

Comment:
What does this rule do? It looks like it only applies when the menu isn't active/visible.